### PR TITLE
Api 5429 dockerize regression tests

### PR DESCRIPTION
**Pull request by lukemarkwordtlibertyits**
The 1st part of a 3 part effort to add the regression tests to the oauth-proxy CI. 

# To run

Build
```sh
docker build . -t vasdvp/lighthouse-oauth-proxy-tests -f Dockerfile.bats
```
Run

```sh
docker run --rm vasdvp/lighthouse-oauth-proxy-tests
# This will give you the instructions to run
```

# To ensure functionality
- [x] test pass against locally running oauth proxy
- [x] test pass agains sandbox oauth proxy

### CODE REVIEW

File: oauth-proxy/Dockerfile.bats [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM vasdvp/health-apis-centos:8
+
+COPY /tests/bats /bats
+COPY /entrypoint_test.sh /bats/entrypoint_test.sh
+
+RUN dnf update -y && \
+    dnf install git -y
+
+RUN dnf install -y -q https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.6-3.3.el7.x86_64.rpm && \
```
Comment (dbrown-va) [inline oauth-proxy/Dockerfile.bats:9]: Should this be a centos 8 package to line up with the base image?

**Review by dbrown-va — COMMENTED** [review]
It looks like the underlying lighthouse-auth-utils runs persist after exiting.

Is it possible to add `--rm` to the calls have it fully cleaned up on the exist?

File: oauth-proxy/Dockerfile.bats (hunk 1 above)
Comment (lukemarkwordtlibertyits) [reply oauth-proxy/Dockerfile.bats]: Just updated

**Comment by lukemarkwordtlibertyits** [comment]
> It looks like the underlying lighthouse-auth-utils runs persist after exiting.
> 
> Is it possible to add `--rm` to the calls have it fully cleaned up on the exist?

done

**Review by dbrown-va — APPROVED** [review]
Confirmed update docker install on centos image.

Built and run locally, looks good.

**Pull request merged**